Clicker/Views/PollBuilder/AskQuestionCell.swift

```diff
@@ -69,14 +98,29 @@ class AskQuestionCell: UICollectionViewCell {
 
 }
 
-extension AskQuestionCell: UITextFieldDelegate {
-
-    func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+extension AskQuestionCell: UITextViewDelegate {
+    
+    func textView(_ textView: UITextView, shouldChangeTextIn range: NSRange, replacementText text: String) -> Bool {
         let maxLength = IntegerConstants.maxQuestionCharacterCount
-        let currentString: NSString = (textField.text ?? "") as NSString
-        let newString: NSString =
-            currentString.replacingCharacters(in: range, with: string) as NSString
+        let currentString: NSString = (textView.text ?? "") as NSString
+        let newString: NSString = currentString.replacingCharacters(in: range, with: text) as NSString
         return newString.length <= maxLength
     }
+    
+    func textViewDidBeginEditing(_ textView: UITextView) {
+        if textView.textColor == UIColor.clickerGrey2 {
+            textView.text = nil
+            textView.textColor = UIColor.black
+        }
+    }
 
+    func textViewDidChange(_ textView: UITextView) {
+        self.charCountLabel.text = "\(questionTextView.text.count)/120"
+        let questionTextViewWidth = contentView.frame.size.width
+        let dynamicSize = textView.sizeThatFits(CGSize.init(width: questionTextViewWidth, height: CGFloat(MAXFLOAT)))
```

Comment:
not sure but this might be worth factoring out into a helper function possibly?

Reply:
+1, could add it as extension to `UITextView`